tools/alpine/Makefile

```diff
@@ -34,8 +43,9 @@ push: hash iid versions.$(ARCH)
 	docker pull $(ORG)/$(IMAGE):$(shell cat hash) || \
 		(docker tag $(shell cat iid) $(ORG)/$(IMAGE):$(shell cat hash) && \
 		 docker push $(ORG)/$(IMAGE):$(shell cat hash))
+
+push-manifest:
 	./push-manifest.sh $(ORG) $(IMAGE)
-	rm -f iid
```

Comment:
I think we still need to delete this on normal push

Reply:
Dang, I thought I had left it. Fixed.